packages/ibm-products/src/components/BigNumbers/utils/enums.js

```diff
@@ -0,0 +1,5 @@
+export const BigNumbersSize = {
```

Comment:
i like the idea here, but i think you can just name this file `constants`. typically for a `utils` folder i'd expect to see utility functions here. if you wanted to move some of those functions here like `truncateValue` and `formatValue` that would be a great way to utilize a utils file and clean up the jsx

Reply:
Yep, I follow. Advise if this needs to be in a directory or if alongside the component in a `utils.js` file will work as I have it now. Formatted that file with constants at the top and utility functions below... seemed to me to clean up the component file some.

Reply:
if you wanted to just create a `constants.js` and import it into the file i think that would be sufficient